src/Tools/ExternalAccess/OmniSharp/Completion/OmniSharpCompletionService.cs

```diff
@@ -31,7 +31,7 @@ internal static class OmniSharpCompletionService
             char? commitCharacter = null,
             bool disallowAddingImports = false,
             CancellationToken cancellationToken = default)
-            => completionService.GetChangeAsync(document, item, completionListSpan, commitCharacter, disallowAddingImports, cancellationToken);
+            => completionService.GetChangeAsync(document, item, commitCharacter, cancellationToken);
```

Comment:
@333fred do you want me to leave the unused parameters?  or should i remove those as well?

Reply:
Just remove the method entirely. We'll move to the public API.